survey-runner.tf

```diff
@@ -106,6 +106,20 @@ module "survey-launcher-for-ecs" {
   alb_listener_rule_priority_offset = 1
 }
 
+module "schema-validator" {
+  source                          = "github.com/ONSdigital/eq-ecs-deploy?ref=create-common-ecs-deploy"
+  env                             = "${var.env}"
+  aws_access_key                  = "${var.aws_access_key}"
+  aws_secret_key                  = "${var.aws_secret_key}"
+  dns_zone_name                   = "${var.dns_zone_name}"
+  ecs_cluster_name                = "${module.eq-ecs.ecs_cluster_name}"
+  aws_alb_listener_arn            = "${module.eq-ecs.aws_alb_listener_arn}"
+  container_name                  = "eq-schema-validator"
+  container_port                  = 5000
+  container_tag                   = "migrate-validation-logic"
+  container_environment_variables = "{\"name\": \"EXAMPLE_VAR\",\"value\": \"TEST\"}"
```

Comment:
Should this be here?

Reply:
Yeh, @ajmaddaford wanted it left as an example.